mapknowledge/apinatomy.py

```diff
@@ -314,6 +315,7 @@ def reclr(blob, start_link):
         layer = []
         col = True
 
+        bads = ('UBERON:0000010', 'UBERON:0001017')  # XXX temp fix
```

Comment:
Shouldn't this list of `bad` identifiers be defined once alongside `EXCLUDED_LAYERS`?